clang/lib/Serialization/ASTWriterStmt.cpp

```diff
@@ -2202,6 +2202,17 @@ void ASTStmtWriter::VisitPackIndexingExpr(PackIndexingExpr *E) {
   Code = serialization::EXPR_PACK_INDEXING;
 }
 
+void ASTStmtWriter::VisitResolvedUnexpandedPackExpr(
```

Comment:
DID you see my comment on how to test these?  I made one at one point, but the idea is you can use PCH to serialize and deserialize, then make sure the ast doesn't change.

Reply:
Yes, and thank you for the heads up. I did add a test that does this.